sources/reference/api/experiment.md

```diff
@@ -199,9 +199,15 @@ property which MUST be a valid [JSON Path][jp]. In addition, the object MAY
 have a `expect` property which is used to compare each value matched by the JSON
 Path to that value. The `expect` property value MUST be a scalar. When the
 `expect` property is not present, the `tolerance` succeeds if the JSON Path
-matched at least one item.
-
-When the `type` property is `"range"`, the object MUST have a `range` 
+matched at least one item. Sometimes the json payload values will be dynamic,
+e.g. field `status` in response payload may provide any values (either `ok` or
+`error` or `info`). Using `expect` field, we can mention only one value as
+expected value but sometimes steady state can be met with one or more values.
+e.g. you want to define two values either `ok` or `info` as expected value.
+In these cases, you can use both `expect` and `expect_one_of` to define both
+expected values.
```

Comment:
I'm still fuzzy about what having the two at the same time achieves. Wouldn't you want one only? Or if you have the two, does one have precedence?